.pre-commit-config.yaml

```diff
@@ -1,5 +1,13 @@
 exclude: (\.git/|\.tox/|\.venv/|le_utils\.egg-info)
 repos:
+-   repo: local
+    hooks:
+    -   id: rebuild
```

Comment:
Does this run `make build` whenever we run `pre-commit run`?

Reply:
Yes, but only if any of the spec JSON files or setup.py have been modified (as per the regex below).